src/any_support.cc

```diff
@@ -144,21 +167,26 @@ LIBRARY_EXPORT int32_t IsAnyOfType(grpc_labview::gRPCid* id, grpc_labview::AnyCl
 //---------------------------------------------------------------------
 //---------------------------------------------------------------------
 LIBRARY_EXPORT int32_t AnyBuilderBegin(grpc_labview::gRPCid** builderId)
-{   
+{
     grpc_labview::InitCallbacks();
 
     auto metadata = std::make_shared<grpc_labview::MessageMetadata>();
     auto rootMessage = new grpc_labview::LVMessage(metadata);
```

Comment:
This codebase is letting C++ exceptions escape from exported C functions.

I'm going to convert this to an issue.

Reply:
https://github.com/ni/grpc-labview/issues/438